WakaTime/WakaTime.m

```diff
@@ -143,12 +143,25 @@ -(void)sendAction:(BOOL)isWrite {
         
         NSMutableArray *arguments = [NSMutableArray array];
         [arguments addObject:[NSHomeDirectory() stringByAppendingPathComponent:WAKATIME_CLI]];
+    
+        NSString* file = self.lastFile;
+        // Handle Playgrounds
+        if ([file.pathExtension isEqual: @"playground"]) {
+            [arguments addObject:@"--project"];
```

Comment:
Agreed. I just found out that wakatime-cli is able to figure out the project for Playgrounds. 
However, like in most cases, Playgrounds are stand-alone projects. So Wakatime will not recognise them as a project themselves. But I guess this is the way to go?